documentation/modules/exploit/unix/http/pfsense_graph_injection_exec.md

```diff
@@ -0,0 +1,117 @@
+## Description
+
+  This module exploits a vulnerability in pfSense version 2.2.6 and before which allows an authenticated user to execute arbitrary operating system commands as root.
+
+## Vulnerable Application
+
+  This module has been tested successfully on version 2.2.6-RELEASE, 2.2.5-RELEASE, and 2.1.3-RELEASE
+
+  Installers:
+
+  * [pfSense 2.2.6-RELEASE](https://nyifiles.pfsense.org/mirror/downloads/old/pfSense-LiveCD-2.2.6-RELEASE-amd64.iso.gz)
+  * [pfSense 2.2.5-RELEASE](https://nyifiles.pfsense.org/mirror/downloads/old/pfSense-LiveCD-2.2.5-RELEASE-amd64.iso.gz)
+  * [pfSense 2.1.3-RELEASE](https://nyifiles.pfsense.org/mirror/downloads/old/pfSense-LiveCD-2.1.3-RELEASE-amd64.iso.gz)
+
+## Verification Steps
+
+  1. Start `msfconsole`
+  2. Do: `use exploit/unix/http/pfsense_graph_injection_exec`
+  3. Do: `set RHOST [IP]`
+  4. Do: `set USERNAME [username]`
+  5. Do: `set PASSWORD [password]`
+  6. Do: `set LHOST [IP]`
+  7. Do: `exploit`
+
+## Scenarios
+
+### pfSense Community Edition 2.2.6-RELEASE
+
+```
+msf exploit(unix/http/pfsense_graph_injection_exec) > options 
```

Comment:
Typically I prefer seeing the options being set ie:
```
use unix/http/pfesense_graph_injection_exec
set rhost ...
set ...
```
That takes less lines and shows any non-defaults being set as opposed to `show options`.  Just for future note